Sources/PT.PM.Common/Nodes/Expressions/ArrayCreationExpression.cs

```diff
@@ -67,6 +67,7 @@ public override int CompareTo(Ust other)
             var otherArrayCreation = (ArrayCreationExpression)other;
 
             int compareKeywordNewResult = (KeywordNew == null && otherArrayCreation.KeywordNew == null) ? 0 : KeywordNew?.CompareTo(otherArrayCreation.KeywordNew) ?? -1;
+
```

Comment:
I mean the string with the code it too long. But I merge it - will fix it later.